### PR TITLE
added workload landing zones

### PR DESCRIPTION
**Pull request by jpalmeiro**

### CODE REVIEW

File: landing-zones/workload_landing_zones/workload_landing_zones.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+&nbsp; 
+
+## 1. What are Workload Landing Zones 
+
+A workload landing zone is an **OCI solution design ready to receieve specific workloads**. Each of these landing zones follow the follows the [tailored landing zone approach](/landing-zones/tailored_landing_zones/tailored_landing_zones.md) and have a design and a runtime, ready for deployment:
+
+- In terms of **design**, each solution will contain the security, network, and runtime views. 
+
+- In terms of **running**, each solution will have a runtime view with **IaC configurations**, ready to be deplyed. Please note that the workload elements (VMs, Databases, etc.) are not deployed.
```
Comment (paalonso) [inline landing-zones/workload_landing_zones/workload_landing_zones.md:15]: Fix typo "deplyed" for "deployed"

File: landing-zones/workload_landing_zones/workload_landing_zones.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+
+## 3. How to Start 
+
+   
+| STEP  | DESCRIPTION |
+|---|---|
+| 1 | Review the design artefact in drawio format.
+| 2 | Configure the runtime view with IaC configurations.
+| 3 | Run IaC configurations with ORM. Your Workload will be on a dedicated stack.
```
Comment (paalonso) [inline landing-zones/workload_landing_zones/workload_landing_zones.md:50]: Should we comment to use ORM OR Terraform CLI?

**Review by paalonso — COMMENTED** [review]
A typo and a comment. The rest looks good to me

File: landing-zones/workload_landing_zones/workload_landing_zones.md (hunk 2 above)
Comment (jpalmeiro) [reply landing-zones/workload_landing_zones/workload_landing_zones.md:50]: we can update to add CLI also

**Review by paalonso — APPROVED** [review]

**Pull request merged**